packages/api/src/main/data-graph/outputs/query-result.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Library } from '../models/library.model'
+import { User } from '../models/user.model'
+
+class QueryResult {
+  public libraries?: Array<Library>
+  public users?: Array<User>
+}
+
+export { QueryResult }
```

Comment:
where is this used? 🤔

Reply:
It'll be used by the messaging infrastructure which is coming soon :)